bokeh/io.py

```diff
@@ -612,32 +615,29 @@ def gridplot(plot_arrangement, **kwargs):
     _push_or_save(grid)
     return grid
 
+@deprecated("Bokeh 0.12.0", "bokeh.models.layouts.Row")
 def hplot(*children, **kwargs):
-    ''' Generate a layout that arranges several subplots horizontally.
-
-    '''
-    _remove_roots(children)
-    layout = HBox(children=list(children), **kwargs)
-    curdoc().add_root(layout)
-    _push_or_save(layout)
+    warnings.warn(
+        """
+        The new Row is responsive by default, it resizes based on the space available.
```

Comment:
suggest:  "Switching to 'Row' will make elements responsive by default (they will resize based on the space available)."